index.html

```diff
@@ -3,9 +3,94 @@
+      <li>SEARCH</li>
+      <li><i class="fa fa-search"></i></li>
+    </ul>
+  </nav>
+  <div class="header">
+    <h1 class="main-title">SURF & PADDLE CO.</h1>
+    <p class="title-sub">An Ironyard Academy Project Est. Fall 2013</p>
+  </div>
+</div>
+<section class="left-section">
+   <h1 class="tit1">A Fantastic Engaging Storyline Goes Here</h1>
+   <img class="icon-section" src="http://placehold.it/25/25" alt="Icon"/>
+   <p class="name1">Chancey Ward</p>
 
-  </body>
+   <p class="underline">on <span>Monday Oct 7th 9:22pm</span></p>
+   <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+   <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+   <h3 class="tit2"> WOAH, THAT'S RADICAL!</h3>
+   <p class="content"> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+   <p class="content"> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in</p>
+   <div class="buttons-left">
+   <button class="first-but"><span>Read Comments</span></button>
```

Comment:
The buttons should be able to have just a single `btn` class or whatever name makes sense. Style them all to be grey and then just make them blue (like the last button in the picture) when the user hovers over them.